green_city/fixtures/create_news.py

```
@@ -60,8 +58,7 @@ def create_not_found_news(auth_token):
     response = requests.post(url, headers=headers, files=files)
     news_id = response.json().get("id")
     delete_url = f"{API_BASE_URL_8085}{ENDPOINTS['delete_eco_news'].format(news_id)}"
-    del_response = requests.delete(delete_url, headers=headers)
-    assert del_response.status_code == 200, "Failed to delete news"
+    requests.delete(delete_url, headers=headers)
```

Pylint (3.3.7)

[warning] Missing timeout argument for method 'requests.delete' can cause your program to hang indefinitely

(W3101)